Libraries/dotNetRdf.Core/Query/Ordering/OrderByVariable.cs

```diff
@@ -102,8 +102,8 @@ public override IEnumerable<string> Variables
         get
         {
             return Child != null
-                ? Variable.AsEnumerable().Concat(Child.Variables).Distinct()
-                : Variable.AsEnumerable();
+                ? ((IList<string>)[Variable, .. Child.Variables]).Distinct()
+                : [Variable];
```

Comment:
Not the nicest syntax when the target type for the collection expression is unknown.